internal/pkg/manifest/workload.go

```diff
@@ -680,7 +680,7 @@ func (s *SecurityGroupsIDsOrConfig) UnmarshalYAML(value *yaml.Node) error {
 
 // GetIDs returns security groups from SecurityGroupsIDsOrConfig that are attached to task.
 // nil is returned if no security groups are specified.
-func (s *SecurityGroupsIDsOrConfig) GetIDs() []string {
+func (s *SecurityGroupsIDsOrConfig) GetIDs() []stringOrFromCFN {
```

Comment:
nit: it's a bit unusual to return a private type from a public method 😅 but I think it's fine since we don't expect `manifest` clients to initialize this type.